docs/basic-features/script.md

```diff
@@ -13,102 +24,141 @@ description: Next.js helps you optimize loading third-party scripts with the bui
 
 </details>
 
-The Next.js Script component enables developers to set the loading priority of third-party scripts to save developer time and improve loading performance.
+The Next.js Script component, [`next/image`](/docs/api-reference/next/script.md), is an extension of the HTML `<script>` element. It enables developers to set the loading priority of third-party scripts to save developer time and improve loading performance.
```

Comment:
Could we clarify how it helps save developer time?

Reply:
```suggestion
The Next.js Script component, [`next/script`](/docs/api-reference/next/script.md), is an extension of the HTML `<script>` element. It enables developers to set the loading priority of third-party scripts to save developer time and improve loading performance.
```